src/applications/pensions/tests/00-all-fields.e2e.spec.js

```diff
@@ -1,993 +0,0 @@
-const moment = require('moment');
-const E2eHelpers = require('../../../platform/testing/e2e/helpers');
-const Timeouts = require('../../../platform/testing/e2e/timeouts');
-const PageHelpers = require('./pensions-helpers');
-const rawTestData = require('./schema/maximal-test.json');
-const FormsTestHelpers = require('../../../platform/testing/e2e/form-helpers');
-
-/**
- * Updates the test data to account for moving dates.
- *
- * Mutates the object. (But sneakily.)
- *
- * The alternative to this function is making the test data a JS file instead of
- * JSON.
- */
-const updateTestData = data => {
-  const d = Object.assign({}, data);
-  d.data.dependents[0].childDateOfBirth = moment()
-    .subtract(20, 'years')
-    .format('YYYY-MM-DD');
-  d.data.dependents[1].childDateOfBirth = moment()
-    .subtract(27, 'years')
-    .format('YYYY-MM-DD');
-  return d;
-};
```

Comment:
It seems that this test originally intended for the dependents' ages to be within or outside of certain ranges for functions such as `isBetween18And23` and `isEligibleForDisabilitySupport`.

The output of those functions changed while the birth dates from the test data remained static. Eventually, those outputs met certain conditions, requiring some fields to be filled that weren't accounted for in the original test data. So this fix to dynamically update the test data was necessary to pass the test.

It seems that at this point in time, the ages are completely out of the ranges of those conditions, so the new Cypress test won't fail in the same way anymore. I will not be updating the ages to match any exact criteria and will leave it for future owners to amend the test data as needed.